backend/app/auth/config.py

```diff
@@ -4,6 +4,7 @@
 JWT_ALGORITHM = os.environ.get("JWT_ALGORITHM", "HS256")
 JWT_SECRET = os.environ.get("JWT_SECRET", "jwt-secret")
 NEXT_PUBLIC_GOOGLE_LOGIN_CLIENT_ID = os.environ.get(
-    "NEXT_PUBLIC_GOOGLE_LOGIN_CLIENT_ID", "update-me"
+    "NEXT_PUBLIC_GOOGLE_LOGIN_CLIENT_ID",
+    "984468254633-hmko6j4sf2jk3l6nd4sugt3e17vjcms8.apps.googleusercontent.com",
```

Comment:
Can we please leave this as "update-me" and add this to template.backend.env?
And can be set in .backend.env

Reply:
Currently, `make run-backend` does not use the docker file. So I put this value here because it is public (easy to see on the front end) and does not change for long time.